changelogs/fragments/827-pam-limits-check-mode.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Please put both fragments in the same file (two items in the `minor_changes` list), and change the number in the filename to 830 (which is this PR's #).